style.css

```diff
@@ -0,0 +1,79 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+}
+
+.wrapper {
+  background-image: url("./images/headerBackgd.png");
+  background-size: cover;
+  width: 100%;
+  max-width: 500px;
+  padding-bottom: 90px;
+  font-family: 'Poppins', sans-serif;
+}
+
+header {
+  background-color: rgb(245, 250, 245);
+  height: auto;
+  display: flex;
+  padding-top: 15px;
+  margin-bottom: 100px;
+  padding-bottom: 10px;
+}
+
+.logo,
+a {
+  font-size: 20px;
+  text-decoration: none;
+  font-weight: bolder;
+  color: #8793ff;
+}
+
+.right-side {
+  flex-grow: 1;
+  margin-left: 20px;
+}
+
+section {
+  display: flex;
+  flex-direction: column;
+  color: rgb(23, 43, 77);
+}
+
+p {
+  font-size: 18px;
+  margin-left: 20px;
+}
```

Comment:
- It would be better to make the font weight of the text lighter to improve your UI design and also to match Figma design. I would recommend importing the `Popping font` with 400 weight so you are able to reduce the weight by using `font-weight: 400;`
<img width="209" alt="image" src="https://user-images.githubusercontent.com/77190203/154270039-9ec4bf6d-9a03-40fe-911d-3c28ae0caa1f.png">